Improve SEO metadata across all docs pages (#397)
powersync-ja/powersync-docs: sync/advanced/compatibility.mdx

---
title: "Compatibility"

> description: "Configure compatibility editions and bucket storage format version for new or legacy PowerSync projects."

Reviewer: ```suggestion
description: "Configure compatibility editions and bucket storage format version in PowerSync's sync config."
```
